--- PKGBUILD	2023-04-14 00:37:54.003009712 +0200
+++ PKGBUILD	2023-04-14 00:35:45.732393132 +0200
@@ -33,8 +33,8 @@
             'd2bc104788290e3de829a232590b66ad745cf5b4762a01acc1191ebf70fef114'
             'f3d4f39ef24e312f610cbb3439fb02bc6b8829e37bcf1a50ae50cd0a69bde5d0'
             'd87d0c9957c613fda272553bee58140349d151ae399f346ddaf6d75ee5916312'
-            ''
-            '')
+            '606e55f06c297d2b508bc4438890b229a1abaa68b0374a2d7f94c8e7be6792d7'
+            '072020e34f2c55b94a9a486829a7eadab0a830ddb4d8e759b0c4e6cf1bde73a6')
 
 prepare() {
   cd "${pkgname}-${pkgver}"
